blog-service/2024-09-20-ui.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Introducing the Sumo Logic New UI
+image: https://help.sumologic.com/img/sumo-square.png
+keywords:
+  - ui
+hide_table_of_contents: true    
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+<a href="https://help.sumologic.com/release-notes-service/rss.xml"><img src={useBaseUrl('img/release-notes/rss-orange2.png')} alt="icon" width="50"/></a>
+
+We're excited to announce the opt-in general availability of our Sumo Logic New UI. This release introduces several enhancements designed to improve your overall experience:
+
+* **Unified Navigation**. Access all Observability features (e.g., Alerts, Log Search, Metrics) and Security features (e.g., Cloud SIEM, Cloud SOAR) through a single, unified left-navigation menu. Unlike the Classic UI, there's no need to navigate away when switching between Observability and Security tools.
+* **Streamlined Administration**. Key administrative functions such as Data Configuration, Help, and User Preferences are now conveniently located in the top toolbar for easier access.
+* **Enhanced Browsing Experience**. We've replaced in-app tabs with native browser tabs to provide a more familiar browsing experience. Enjoy session history, back button support, favorites, and other native browser functionalities.
+* **Improved Performance**. The simplified UI boosts performance and usability, offering a smoother and faster browsing experience that enables quicker incident resolution. Significant improvements include:
+   * **First Contentful Paint (FCP)**. ~90% improvement (synthetic), 75% improvement (RUM).
+   * **Time to Interactivity (TTI)**. ~27% improvement (synthetic), 50% improvement (RUM).
+* **Stateful URLs**. URLs now reflect real-time changes made within the UI. This ensures that any modifications are immediately reflected in the URL parameters, allowing for easy sharing of the latest content with your team.
+
+Explore these improvements in our [documentation](/docs/get-started/sumo-logic-ui) and this [click-through demo](https://app.storylane.io/share/dbjtjmulqvcb). We value your feedback as we continue to refine our UI.
```

Comment:
I think we should remove the link to the click-through demo, since it's a little stale.

Reply:
Good thinking! Done.